UnityProject/Assets/Scripts/Core/UniversalObjectPhysics.cs

```diff
@@ -176,6 +178,21 @@ public ObjectContainer ContainedInContainer
 
 	[PlayModeOnly] public Vector2 newtonianMovement; //* attributes.Size -> weight
 
+
+	public Vector2 SetNewtonianMovement
```

Comment:
It seems misleading to prefix a property with `Set`. Why not just call it `NewtonianMovement`?

If the name is too similar to `newtonianMovement`, the convention is to prefix the backing field with an underscore.

Reply:
it's the one you want to use when you want to set newtonianMovement,   Since it has the cap on it

Reply:
It's a property, so you can read from it too, in that case `Set` makes no sense. I assume `newtonianMovement` should never exceed the `MAX_SPEED`, so making it private and only using the property seems better?

Reply:
yeah, that's exactly what properties are for. Make the field private and wrap it with the property which has the cap when you set a value to it, return the private field when you get it.